jwst/jump/tests/test_miri_first_lastframe.py

```diff
@@ -0,0 +1,82 @@
+"""
+Unit test for jump step to test not using MIRI first frame or last frame if set to DO_NOT_USE
+"""
+import pytest
+import numpy as np
+from jwst.jump import JumpStep
+from jwst import datamodels
+from jwst.datamodels import dqflags, MIRIRampModel
+from jwst.pipeline.collect_pipeline_cfgs import collect_pipeline_cfgs
+
+
+@pytest.fixture(scope='function')
+def science_image():
+    """Generate science image"""
+    # size of integration
+    nints = 1
+    ngroups = 7
+    xsize = 1032
```

Comment:
Don't really need to be testing this many pixels that all have identical values (1 would be sufficient). I suggest reducing to something like xsize=25, ysize=25, at most.